docs/How-To/farming_plans.md

```diff
@@ -0,0 +1,243 @@
+  "description": "Are you ready to farm?",
+  "add_request_proposals": [
+    {
+      "name": "First Public Farming Plan",
+      "farming_pool_address": "cosmos1zaavvzxez0elundtn32qnk9lkm8kmcszzsv80v",
+      "termination_address": "cosmos1zaavvzxez0elundtn32qnk9lkm8kmcszzsv80v",
+      "staking_coin_weights": [
+        {
+          "denom": "PoolCoinDenom",
+          "amount": "1.000000000000000000"
+        }
+      ],
+      "start_time": "2021-07-15T08:41:21.662422Z",
+      "end_time": "2022-07-16T08:41:21.662422Z",
+      "epoch_amount": [
+        {
+          "denom": "uatom",
+          "amount": "1"
+        }
+      ]
+    },
+    {
+      "name": "First Public Farming Plan",
+      "farming_pool_address": "cosmos1zaavvzxez0elundtn32qnk9lkm8kmcszzsv80v",
+      "termination_address": "cosmos1zaavvzxez0elundtn32qnk9lkm8kmcszzsv80v",
+      "staking_coin_weights": [
+        {
+          "denom": "PoolCoinDenom",
+          "amount": "1.000000000000000000"
+        }
+      ],
+      "start_time": "2021-07-15T08:41:21.662422Z",
+      "end_time": "2022-07-16T08:41:21.662422Z",
+      "epoch_ratio": "1.000000000000000000"
+    }
+  ]
+}
+```
+
+## UpdatePublicFarmingFixedAmountPlan
+
+```json
+{
+  "title": "Let's Update the Farming Plan 1",
+  "description": "FarmingPoolAddress needs to be changed",
+  "update_request_proposals": [
+    {
+      "plan_id": 1,
+      "farming_pool_address": "cosmos13w4ueuk80d3kmwk7ntlhp84fk0arlm3mqf0w08",
+      "termination_address": "cosmos13w4ueuk80d3kmwk7ntlhp84fk0arlm3mqf0w08",
+      "staking_coin_weights": [
+        {
+          "denom": "uatom",
+          "amount": "1.000000000000000000"
+        }
+      ],
+      "start_time": "2021-07-15T08:41:21.662422Z",
+      "end_time": "2022-07-16T08:41:21.662422Z",
+      "epoch_amount": [
+        {
+          "denom": "uatom",
+          "amount": "1"
+        }
+      ]
+    }
+  ]
+}
+```
+
+## DeletePublicFarmingFixedAmountPlan
+
+```json
+{
+  "title": "Delete Public Farming Plan 1",
+  "description": "This plan is no longer needed",
+  "delete_request_proposals": [
+    {
+      "plan_id": 1
+    }
+  ]
+}
+```
+
+# Private Farming Plan
+
+## PrivateFarmingFixedAmountPlan
+
+```json
+{
+	"name": "This Farming Plan intends to incentivize ATOM HODLERS!",
+  "staking_coin_weights": [
+	  {
+	      "denom": "uatom",
+	      "amount": "1.000000000000000000"
+	  }
+  ],
+  "start_time": "2021-07-15T08:41:21.662422Z",
+  "end_time": "2022-07-16T08:41:21.662422Z",
+  "epoch_amount": [
+    {
+      "denom": "uatom",
+      "amount": "1"
+    }
+  ]
+}
+```
+
+## PrivateFarmingRatioPlan
+
+```json
+{
+	"name": "This Farming Plan intends to incentivize ATOM HODLERS!",
+  "staking_coin_weights": [
+	  {
+	      "denom": "uatom",
+	      "amount": "1.000000000000000000"
+	  }
+  ],
+  "start_time": "2021-07-15T08:41:21.662422Z",
+  "end_time": "2022-07-16T08:41:21.662422Z",
+  "epoch_ratio":"1.000000000000000000"
+}
+```
```

Comment:
Could you add CLI Commands of https://gist.github.com/kogisin/258a8379de061bf8f84a1f82effd2367#gistcomment-3829971 also? 
And some json examples need to sort indent.

Reply:
I'll add the command-line interfaces and sort the json examples.